src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.module.css

```diff
@@ -18,10 +18,9 @@
         font-weight: 700;
         line-height: 2.8rem;
         overflow: hidden;
-        white-space: wrap;
```

Comment:
This property doesn't exist, I changed it to the correct name in my previous PR.